### PR TITLE
Fix docker upload bugs

### PR DESCRIPTION
**Pull request by dhuangnm**
- Fixed a bug due to output whl and build version value missing
- Renamed input argument wheel to whl to be consistent across actions and jobs
- Removed unnecessary part in Dockerfile that caused image build to fail due to permission issue

### CODE REVIEW

File: .github/actions/nm-get-docker-tags/action.yml [hunk 1]
```diff
@@ -22,15 +22,15 @@ runs:
   steps:
   - id: tags
     run: |
-      BUILD_VERSION=`echo "${{ inputs.wheel }}" | cut -d'-' -f2`
+      BUILD_VERSION=`echo "${{ inputs.whl }}" | cut -d'-' -f2`
       if [[ "${{ inputs.wf_category }}" == "RELEASE" ]]; then
-          TAG="v${build_version}"
+          TAG="v${BUILD_VERSION}"
           EXTRA_TAG=latest
```
Comment (derekk-nm) [inline .github/actions/nm-get-docker-tags/action.yml:28]: should the EXTRA_TAG value be in quotes?

File: .github/actions/nm-get-docker-tags/action.yml [hunk 2]
```diff
@@ -22,15 +22,15 @@ runs:
...
       else
-          TAG=`echo "${build_version}" | cut -d'.' -f4`
+          TAG=`echo "${BUILD_VERSION}" | cut -d'.' -f4`
           EXTRA_TAG=nightly
       fi
-      if [[ "${{ inputs.wheel }}" == "latest" ]]; then
+      if [[ "${{ inputs.whl }}" == "latest" ]]; then
```
Comment (derekk-nm) [inline .github/actions/nm-get-docker-tags/action.yml:33]: why is this statement necessary?  it seems that line 25 would have already set `BUILD_VERSION="latest"`

File: .github/actions/nm-get-docker-tags/action.yml (hunk 1 above)
Comment (dhuangnm) [reply .github/actions/nm-get-docker-tags/action.yml:28]: This should work fine in bash.

File: .github/actions/nm-get-docker-tags/action.yml (hunk 2 above)
Comment (dhuangnm) [reply .github/actions/nm-get-docker-tags/action.yml:33]: This is in the case when the whl is passed in as 'latest' and line 25 doesn't extract the expected version so we need to make sure build_version is set to "latest".

**Review by derekk-nm — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -62,20 +62,6 @@ RUN apt-get update -y && apt-get install -y ccache
...
-ENV FLASH_ATTN_VERSION=${flash_attn_version}
-
-WORKDIR /usr/src/flash-attention-v2
-
-# Download the wheel or build it if a pre-compiled release doesn't exist
-RUN pip --verbose wheel flash-attn==${FLASH_ATTN_VERSION} \
-    --no-build-isolation --no-deps --no-cache-dir
-
-#################### FLASH_ATTENTION Build IMAGE ####################
-
```
Comment (mgoin) [inline Dockerfile:78]: This is in the upstream docker, can we keep this?

**Review by mgoin — CHANGES_REQUESTED** [review]
I'm concerned about removing flash attention as it is in the upstream image

**Review by mgoin — APPROVED** [review]
Unblocking, thanks for discussion

**Comment by dhuangnm** [comment]
> Unblocking, thanks for discussion

Thanks Michael!

**Review by andy-neuma — APPROVED** [review]
thanks

**Pull request merged**